### PR TITLE
Merge OSS DB WTH Oracle Update and other changes 

### PR DESCRIPTION
**Pull request by perktime**
This is a major update for the OSS DB migration WTH

### CODE REVIEW

File: 033-OSSDatabaseMigration/Coach/01-discovery.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Challenge 1: Assessment
```
Comment (Whowong) [inline 033-OSSDatabaseMigration/Coach/01-discovery.md:1]: Does the title of the challenge need to change as well?  I see the file and challenge named differently on the readme page.

**Review by Whowong — CHANGES_REQUESTED** [review]
Just one comment about the coach challenge 1 title.  Other than that, looks awesome.

File: 033-OSSDatabaseMigration/Student/03-offline-cutover-validation.md [hunk 2]
```diff
@@ -53,12 +53,58 @@ Status field changes from "Terminating" to "ContainerCreating" and then to "Runn
 
 ```
 
+* Oracle:
+
+You will need to change the database type in the values-oracle.yaml file from "oracle" to "postgres" before you run these steps.
```
Comment (jordanbean-msft) [inline 033-OSSDatabaseMigration/Student/03-offline-cutover-validation.md:58]: `values-oracle.yaml`

File: 033-OSSDatabaseMigration/Student/03-offline-cutover-validation.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 
 ## Introduction
- Reconfigure the application to use the appropriate connection string that uses Azure DB and validate that the application is working
+ Reconfigure the application to use the appropriate connection string that uses Azure DB and validate that the application is working. You have to do this by redeploying the Pizzeria container application(s). 
 
 ## Description
-You will reconfigure the application to use a connection string that points to the Azure DB for PostgreSQL/MySQL. You will need to update the ContosoPizza/values-mysql.yaml or ContosoPizza/values-postgresql.yaml values file with the updated values for dataSourceURL, dataSourceUser and dataSourcePassword using the appropriate Azure DB values for PostgreSQL/MySQL:
+You will reconfigure the application to use a connection string that points to the Azure DB for PostgreSQL/MySQL. You will need to update the ContosoPizza/values-mysql.yaml, ContosoPizza/values-postgresql.yaml and/or ContosoPizza/values-oracle.yaml values file(s) with the updated values for dataSourceURL, dataSourceUser and dataSourcePassword using the appropriate values for Azure DB for PostgreSQL/MySQL:
```
Comment (jordanbean-msft) [inline 033-OSSDatabaseMigration/Student/03-offline-cutover-validation.md:9]: `ContosoPizza/values-msql.yaml` use the backticks to format code or file references

File: 033-OSSDatabaseMigration/Student/06-private-endpoint.md [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
 
 ## Description
-You will add a private endpoint for Azure DB for PostgreSQL/MySQL
-You will reconfigure the application to use a connection string that points to the private IP address for Azure DB for PostgreSQL/MySQL. You will need to update the ContosoPizza/values-mysql.yaml or ContosoPizza/values-postgresql.yaml values file with the updated values for dataSourceURL, dataSourceUser and dataSourcePassword using the appropriate Azure DB values for PostgreSQL/MySQL:
+You will add a private endpoint for Azure DB for PostgreSQL/MySQL Single Server
+You will reconfigure the application to use a connection string that points to the private IP address for Azure DB for PostgreSQL/MySQL Single Server. You will need to update the ContosoPizza/values-mysql.yaml or ContosoPizza/values-postgresql.yaml values file with the updated values for dataSourceURL, dataSourceUser and dataSourcePassword using the appropriate Azure DB values for PostgreSQL/MySQL:
```
Comment (jordanbean-msft) [inline 033-OSSDatabaseMigration/Student/06-private-endpoint.md:11]: `ContosoPizza/values-mysql.yaml`

File: 033-OSSDatabaseMigration/Coach/01-discovery.md (hunk 1 above)
Comment (perktime) [reply 033-OSSDatabaseMigration/Coach/01-discovery.md:1]: Fixed

File: 033-OSSDatabaseMigration/Student/03-offline-cutover-validation.md (hunk 2 above)
Comment (perktime) [reply 033-OSSDatabaseMigration/Student/03-offline-cutover-validation.md:58]: backticks added

File: 033-OSSDatabaseMigration/Student/06-private-endpoint.md (hunk 4 above)
Comment (perktime) [reply 033-OSSDatabaseMigration/Student/06-private-endpoint.md]: added backticks

**Review by jordanbean-msft — APPROVED** [review]

**Review by Whowong — APPROVED** [review]

**Pull request merged**